CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 ### Fixed
 - Fixed detector `DontUseFloatsAsLoopCounters` to prevent false positives. ([#2126](https://github.com/spotbugs/spotbugs/issues/2126))
 - Fixed regression in `4.7.2` caused by ([#2141](https://github.com/spotbugs/spotbugs/pull/2141))
+- improve compatibility with later version of jdk (>= 13). ([#2188](https://github.com/spotbugs/spotbugs/issues/2188))
```

Comment:
Should start with a capital letter.

Reply:
I agree, but it is in the `master` branch. I just merged this file by putting a line after this one. (And the line I added starts with capital letter.) I would better not touch it in this particular PR.